packages/core/src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { Theme, css } from '@emotion/react';
+import Button from '@components/Button';
+import Typography from '@components/Typography';
+import { ButtonGroupProps, Item } from './types';
+
+const ButtonItem = (theme: Theme) => css`
+  height: auto;
+  padding: 12px;
+  border-radius: 0;
+  box-shadow: none;
+  user-select: none;
+
+  &:hover,
+  &:focus,
+  &:active {
+    box-shadow: none;
+  }
+
+  &:first-of-type {
+    border-radius: 6px 0 0 6px;
+  }
+
+  &:last-child {
+    border-radius: 0 6px 6px 0;
+  }
+
+  &:not(:last-child) {
+    margin-right: 1px;
+  }
+
+  &.active {
+    background: ${theme.colors.greyFocused};
+  }
+`;
+
+export const ButtonGroup = ({
+  items,
+  cssStyles,
```

Comment:
I don't like this name. We don't need to mention this is a `CSS` style. It's obvious. It's better to show that this style relates to the button component.

Reply:
done